package.json

```diff
@@ -38,11 +38,11 @@
     "rollup-plugin-serve": "^2.0.2",
     "rollup-plugin-string": "^3.0.0",
     "rollup-plugin-terser": "^7.0.2",
-    "ts-jest": "^26.4.4",
-    "ts-node": "^9.0.0",
-    "tslib": "^2.0.3",
-    "typedoc": "^0.25.4",
-    "typescript": "^4.3.5"
+    "ts-jest": "^29.1.2",
+    "ts-node": "^10.9.2",
+    "tslib": "^2.6.2",
+    "typedoc": "^0.25.13",
+    "typescript": "^5.4.5"
```

Comment:
I think three-vrm still does not require users to use typescript v5 yet after this PR.

Reply:
PR to remove `downlevel-dts` https://github.com/pixiv/three-vrm/pull/1302 . I think that is still valid.